README.md

```diff
@@ -1,3 +1,9 @@
+<p align="center">
+🚀 Update Y2024: Just launched <b><a href="https://www.rudderstack.com/use-case/identity-resolution/">RudderStack Profiles</a></b>
+ <br/>Scalable identity resolution in your warehouse for a complete picture of your customer
```

Comment:
@gitcommitshow do we want to tweak the messaging a bit to "for a 360 view of your customer" - so it's aligned with the other marketing materials across our website and blogs as well?

Reply:
Done